src/pages/publication/PublicationFormValidationSchema.ts

```diff
@@ -83,6 +83,14 @@ const chapterPublicationInstance = {
   type: Yup.string().length(0),
 };
 
+const journalPublicationContext = {
+  title: Yup.string().required(ErrorMessage.REQUIRED),
+};
+
+const degreeAndReportPublicationContext = {
+  publisher: Yup.string().required(ErrorMessage.REQUIRED),
```

Comment:
Hva er egentlig `publisher` som ligger i `publicationContext` nå?
Vi har ett annet `publisher`-felt utenfor `entityDescription` som bare skal settes av backend og er customer institution til publikasjonens eier (tror jeg). Det er ingen sammenheng mellom disse?

Reply:
Heheheh, bra du også reagerer på dette, for det gjorde jeg også 😝 For typene Degree og Report (og Book tror jeg), så har man en `publisher` (utgiver) assosiert med publikasjonen, mens for Journal har man en `title` (tittel på tidsskrift) assosiert med publikasjonen.

`publisher` inni `publicationContext` har ingenting å gjøre med `publisher` utenfor `entityDescription`, nei.

Se på de forskjellige typene her hvis du er interessert: https://github.com/BIBSYSDEV/nva-datamodel-java/tree/develop/documentation

Reply:
jaha, skjønner ikke helt den men :p

Reply:
nei, ikke jeg heller, men sånn er det tydeligvis 😅